src/commands/mod.rs

```diff
@@ -150,27 +150,27 @@ impl fmt::Display for RbfErrorInfo {
     }
 }
 
-/// A wallet transaction getter which fetches the transaction from our Bitcoin backend with a cache
+/// A wallet transaction getter which fetches the transaction from our database backend with a cache
```

Comment:
Perhaps I could keep the Bitcoin backend as a fallback in case we can't find the tx in the DB.